If Helm <3.0 is installed, apiVersion is not valid.

**Describe the bug**
I'm following the install process in the readme. `make skaffold` fails with an error.

**To Reproduce**
Steps to reproduce the behavior:
1. `make skaffold`
2. Receive error below.

**Expected behavior**
Yinyo would be installed

**Screenshots or logs**
```
Successfully built 380c8ea2f85c
Successfully tagged openaustralia/yinyo-scraper:v0.9-beta.2-90-g0d20ea9-dirty
Tags used in deployment:
 - openaustralia/yinyo-server -> openaustralia/yinyo-server:a699e6361044655d6975f959650cdb7d374fc3dd97831f4df6b65a9591118d8e
 - openaustralia/yinyo-scraper -> openaustralia/yinyo-scraper:380c8ea2f85c2d890b14c937f86e220a2fcf530cbd57c115cdceb463724f3616
   local images can't be referenced by digest. They are tagged and referenced by a unique ID instead
Starting deploy...
Helm release yinyo not installed. Installing...
Error: apiVersion 'v2' is not valid. The value must be "v1"
Cleaning up...
Error: could not find tiller
Error: could not find tiller
Error: could not find tiller
FATA[0412] exiting dev mode because first deploy failed: deploying yinyo: building helm dependencies: exit status 1
make: *** [Makefile:44: skaffold] Error 1
```

**Additional context**
```* minikube 1.8.2 is available! Download it: https://github.com/kubernetes/minikube/releases/tag/v1.8.2```
`! minikube v1.6.2 on Ubuntu 19.10`
`* Preparing Kubernetes v1.17.0 on Docker '19.03.5' ...`

--- a/README.md
+++ b/README.md
@@ -45,7 +45,8 @@ a guide for people who are just interested in being users of the API.
 ### Main dependencies
 
 - [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/)
-- [Helm](https://helm.sh/docs/intro/install/)
+- [Helm >=3.0](https://helm.sh/docs/intro/install/)
+  - _**nb:** Helm is still releasing updates to 2.x; be sure to install the latest 3.x, not just the latest release_
 - [Skaffold](https://skaffold.dev/docs/quickstart/)
 - [Go 1.13](https://golang.org/dl/)
 

